src/style.css

```diff
@@ -1,3 +1,77 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  --header-min-width: 1024px;
+  --header-max-width: 1200px;
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  min-width: var(--header-min-width);
+  max-width: var(--header-max-width);
+  background-color: #fff;
+}
+
+.logo__img {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
```

Comment:
line-height: 14px;
  text-align: left;
  text-decoration: none;
  text-transform: uppercase;

this styles do nothing
text-decoration and text-transform you already added in .nav__link